Cargo.toml

```diff
@@ -7,7 +7,9 @@ exclude = [
 resolver = "2"
 
 members = [
-    "crypto",
+    "crypto/sphinx",
+    "crypto/random",
+    "crypto/types",
```

Comment:
Would this be better called `crypto/primitives`?

Reply:
The term "primitive" has different meaning in cryptography.